### DISCUSSION TITLE
Why does the ente-io/ente repo contain releases for both Auth and Photos apps? Do they share the same codebase?

### DISCUSSION THREAD

**Discussion by quinncomendant** [ente-io/ente#1867] [General]
I notice the ente-io/ente repository [releases page](https://github.com/ente-io/ente/releases) lists release for both Auth and Photos. I'm confused why they're together in the same repo, since they're different apps. Why? 🤔

I'm considering to start using the Auth app, and was looking for the codebase for it but couldn't find it. Is it part of the same codebase as the Photos app?

It looks like great software, thanks!

**Comment by sv3nnie** [comment]
Yes, ente uses a monorepo. This repo contains all the source code for both products. The source code for the auth app can be found in the [auth](https://github.com/ente-io/ente/tree/main/auth) folder.
